### ISSUE TITLE
Horizontal split not working

### ISSUE THREAD

**Issue by scottballentine** [MartinPacker/md2pptx#80]
When using `ContentSplitDirection: horizontal`, get an error:
```
Traceback (most recent call last):
  File "md2pptx", line 3877, in <module>
    slideNumber, slide, sequence = createSlide(prs, slideNumber, slideInfo)
  File "md2pptx", line 2603, in createSlide
    slide = createContentSlide(
  File "md2pptx", line 1584, in createContentSlide
    renderingRectangle = Rectangle(blockTop, blockLeft, blockHeight, blockWidth)
UnboundLocalError: local variable 'blockLeft' referenced before assignment
```
I think I see what's missing... will try a fix shortly.

**Comment by MartinPacker** [comment]
Reproduced and fixed in Test.

**Comment by MartinPacker** [comment]
Fixed in v2.2 and shipped.
